gatsby-config.js

```diff
@@ -209,7 +209,34 @@ module.exports = {
     {
       resolve: 'gatsby-plugin-sitemap',
       options: {
+        output: `/sitemap-index`,
         excludes: DISLALLOW_LIST,
```

Comment:
just saw this. Can you rename

Reply:
afaik it should match the other sitemaps, which are currently only on sitemap-index. sitemap-index should collect all sitemaps though, so I'll change and see what happens

Reply:
Meant the type here. DIESEL-ALLOW